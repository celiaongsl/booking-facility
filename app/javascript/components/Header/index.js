import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import { Typography } from "@material-ui/core";

import AppBar from "@material-ui/core/AppBar";
import Toolbar from "@material-ui/core/Toolbar";

const useStyles = makeStyles(() => ({
  AppBarRoot: {
    backgroundColor: "white",
  },
}));

const Header = () => {
  const classes = useStyles();

  return (
    <AppBar position="static" className={classes.AppBarRoot}>
      <Toolbar>
        <Typography variant="h5" color="primary">Book a Facility</Typography>
      </Toolbar>
    </AppBar>
  );
};

export default Header;
