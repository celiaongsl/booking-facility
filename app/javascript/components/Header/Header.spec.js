import React from "react";
import Header from "./";
import { render } from "@testing-library/react";

describe("<Header />", () => {
  it("renders the component", () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
