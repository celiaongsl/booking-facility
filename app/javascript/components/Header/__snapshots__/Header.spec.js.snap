// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders the component 1`] = `
<DocumentFragment>
  <header
    class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary makeStyles-AppBarRoot-1 MuiPaper-elevation4"
  >
    <div
      class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h5
        class="MuiTypography-root MuiTypography-h5 MuiTypography-colorPrimary"
      >
        Book a Room
      </h5>
    </div>
  </header>
</DocumentFragment>
`;
